ros_packages/motors/pib_motors/pib_motors/motor.py

```diff
@@ -8,18 +8,22 @@ class Motor:
 	MIN_ROTATION: int = -9000
 	MAX_ROTATION: int = 9000
 
+	NO_CURRENT: int = -1
+
 	def __init__(self, name: str, bricklet_pins: list[BrickletPin], invert: bool):
 		self.name: str = name
 		self.visible: bool = True
 		self.bricklet_pins = bricklet_pins
 		self.invert: bool = invert
-		self.rotation_range_min: int = self.MIN_ROTATION
-		self.rotation_range_max: int = self.MAX_ROTATION
+		self.rotation_range_min: int = Motor.MIN_ROTATION
```

Comment:
Why no self. any more?

Reply:
because 'MIN_ROTATION' and 'MAX_ROTATION' are static fields of the 'Motor' class and should therefore be accessed via the class directly (although the 'self.' variant would also work)